packages/web/src/components/withCellHOC.tsx

```diff
@@ -26,7 +26,6 @@ export type CellSuccessStateComponent =
 
 export interface WithCellProps {
   beforeQuery?: <TProps>(props: TProps) => { variables: TProps }
-  // @ts-expect-error We do not know, and even really care, what they are here.
```

Comment:
this was throwing an "unused directive" error for me so I removed it -- please let me know if you want it back in

Reply:
looks like this caused some issues with the automated lintings/builds; reverted

Reply:
You can tighten up the typing, that way you can remove the comment
```
  QUERY: DocumentNode | ((variables: Record<string, unknown>) => DocumentNode)
```

Reply:
good call -- updated